.envrc.example

```diff
@@ -15,3 +15,5 @@ export SQLSERVER_CONNECTION_NAME="project:region:instance"
 export SQLSERVER_USER="sqlserver-user"
 export SQLSERVER_PASS="sqlserver-password"
 export SQLSERVER_DB="sqlserver-db-name"
+
+export GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json
```

Comment:
Does this actually need to be set, or can we fake this env var for that specific test?

Reply:
For the e2e test to work, we'll need this.

Reply:
It's also already present in our CI environment.